docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/FastForestWithOptions.cs

```diff
@@ -9,19 +9,23 @@ namespace Samples.Dynamic.Trainers.Regression
 {
     public static class FastForestWithOptionsRegression
     {
-        // This example requires installation of additional NuGet package
-        // <a href="https://www.nuget.org/packages/Microsoft.ML.FastTree/">Microsoft.ML.FastTree</a>. 
+        
```

Comment:
extra line here too?

Reply:
Yep @RadicalRayan explained why below. I think he's asking Zeeshan about the feasibility of removing the line from all of the tt files with the NuGet package installation comment.